nengo_loihi/neurons.py

```diff
@@ -8,6 +8,41 @@
 from nengo.params import NumberParam
 
 
+def loihi_lif_rates(neuron_type, x, gain, bias, dt):
+    # discretize tau_ref as per CxGroup.configure_lif
+    tau_ref = dt * np.round(neuron_type.tau_ref / dt)
+
+    j = neuron_type.current(x, gain, bias) - 1
+
+    out = np.zeros_like(j)
+    p = tau_ref + neuron_type.tau_rc * np.log1p(1. / j[j > 0])
```

Comment:
What is `p`? In generally I'm really not a big fan of one-letter variable names as they're hard to grep for, though I am ok for well known names like `j` for current.

Reply:
It's the period.